README.md

```diff
@@ -17,7 +17,8 @@ and [ppx_tools/rewriter](https://github.com/ocaml-ppx/ppx_tools/blob/master/rewr
 - compiler-libs.common
 - fswatch (for Build + Watch)
 - ANSITerminal
-- ppx_expect (for testing)
+- ppx_expect >= 0.13.1
```

Comment:
we said below the ppx_expect is only for testing... Is this the case or should I change it to be a build dependency too?

Reply:
hmm. Lets leave it as build dependency. It seems to pass the checks online.

Reply:
wait it's currently a test dependency i mean, but the readme says it's build dep

Reply:
oh gotcha. Uhh, I'll add back your change for the testing note, since it's not strictly build

Reply:
sounds good